reader/traceql/transpiler/clickhouse_transpiler/attr_condition.go

```diff
@@ -105,15 +105,10 @@ func (a *AttrConditionPlanner) aggregator(main sql.ISelect) error {
 		return nil
 	}
 
-	if strings.TrimPrefix(a.AggregatedAttr, "span.") != a.AggregatedAttr {
-		a.AggregatedAttr = a.AggregatedAttr[5:]
-	}
-	if strings.TrimPrefix(a.AggregatedAttr, "resource.") != a.AggregatedAttr {
-		a.AggregatedAttr = a.AggregatedAttr[9:]
-	}
-	if strings.TrimPrefix(a.AggregatedAttr, ".") != a.AggregatedAttr {
-		a.AggregatedAttr = a.AggregatedAttr[1:]
-	}
+	a.AggregatedAttr = strings.TrimPrefix(a.AggregatedAttr, "span.")
```

Comment:
Same issue. I believe it won't work for the edge cases. Let's revert to the initial code.